chain/consensus/platform/platform.go

```diff
@@ -0,0 +1,54 @@
+package platform
```

Comment:
Out of a curiosity, why platform? :)

Reply:
That's a good name I learned from ArdanLabs training ;-)